src/components/navBar/navBar.js

```diff
@@ -1,36 +1,166 @@
+import styles from "@/styles/header.module.scss";
+import logo from "../../../public/logoWithText.svg";
+import facebook from "../../../public/facebook.png";
+import instagram from "../../../public/instagram.png";
+import twitter from "../../../public/twitter.png";
+
 
 const MENU_LIST = [
-  { text: "Home", href: "/" },
-  { text: "About Us", href: "/about" }
+  { text: "Öppettider & Kontakt", href: "/" },
+  { text: "Om oss", href: "/about" },
+  { text: "Biljettinfo", href: "/" },
+  { text: "Nyhetsbrev", href: "/" },
+  { text: "Presentkort", href: "/" }
+];
+
+const SUB_MENU_LIST = [
+  {
+    text: "Repertoar",
+    content: [
+      { text: "Hela bioprogrammet", href: "/" },
+      { text: "Kommande premiärer", href: "/" },
+      { text: "Erbjudande på matinéfilm", href: "/" }
+    ]
+  },
+  {
+    text: "Serverade salonger",
+    content: [
+      { text: "Salong Alcatraz", href: "/" },
+      { text: "Salong Bar Deco", href: "/" },
+      { text: "Salong Camera", href: "/" }
+    ]
+  },
+  {
+    text: "Specialvisningar",
+    content: [
+      { text: "Klassiker", href: "/" },
+      { text: "Sing Along", href: "/" },
+      { text: "Den Dåliga Filmklubben", href: "/" },
+      { text: "Frukostbio", href: "/" },
+      { text: "Cinemateket", href: "/" },
+      { text: "Nyår på Tomtens Biograf", href: "/" },
+      { text: "Hollywood om Hollywood", href: "/" }
+    ]
+  },
+  {
+    text: "Live på bio",
+    content: [
+      { text: "Hela programmet", href: "/" },
+      { text: "Opera", href: "/" },
+      { text: "Teater", href: "/" },
+      { text: "Konst", href: "/" },
+      { text: "Konsert", href: "/" }
+    ]
+  },
+  {
+    text: "Bar & Bistro",
+    content: [
+      { text: "Menyn", href: "/" },
+      { text: "Premiärfredag", href: "/" },
+      { text: "Filmquiz", href: "/" }
+    ]
+  }
 ];
 
 const Navbar = ({setShowLogInModal}) => {
   const [navActive, setNavActive] = useState(null);
   const [activeIdx, setActiveIdx] = useState(-1);
 
+  const [openNav, setOpenNav] = useState(false);
+  const [showSearchBox, setShowSearchBox] = useState(false);
+
   return (
-    <header>
-      <nav>
-        <div onClick={() => setNavActive(!navActive)}></div>
-        <div className={`${navActive ? "active" : ""} nav__menu-list`}>
-          {MENU_LIST.map((menu, idx) => (
-            <div
-              onClick={() => {
-                setActiveIdx(idx);
-                setNavActive(false);
-              }}
-              key={menu.text}
+    <>
+      <header className={styles.header}>
+        <Link href="/">
+          <Image
+            className={styles.headerLogo}
+            role="button"
+            src={logo}
+            alt="logo"
+          />
+        </Link>
+        <button
+          onClick={() => {
+            setOpenNav(!openNav);
+          }}
+          className={`${styles.headerListToggle} ${
+            openNav ? styles.open : null
+          }`}
+        ></button>
```

Comment:
When you have an empty element (with no children), I usually prefer to use self-closing tags, i.e. `<button/>` instead of `<button></button>`